File: app/src/main/java/com/zs/login/myvarioustest/activity/ItemActivity.java
package com.zs.login.myvarioustest.activity;

import android.app.Activity;
import android.graphics.Color;
import android.os.Build;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.view.View;

import com.zs.login.myvarioustest.R;
import com.zs.login.myvarioustest.view.ItemView;

import butterknife.Bind;
import butterknife.ButterKnife;

/**
 * Created by zs
 * Date：2017年 05月 19日
 * Time：15:58
 * —————————————————————————————————————
 * About:
 * —————————————————————————————————————
 */

public class ItemActivity extends Activity implements View.OnClickListener{

    @Bind(R.id.item_1)
    ItemView itemView1;
    @Bind(R.id.item_4)
    ItemView itemView4;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.item_layout);
        ButterKnife.bind(this);

        if (Build.VERSION.SDK_INT >= 21) {
            View decorView = getWindow().getDecorView();
            int option = View.SYSTEM_UI_FLAG_LAYOUT_HIDE_NAVIGATION
                    | View.SYSTEM_UI_FLAG_LAYOUT_FULLSCREEN
                    | View.SYSTEM_UI_FLAG_LAYOUT_STABLE;
            decorView.setSystemUiVisibility(option);
            getWindow().setNavigationBarColor(Color.TRANSPARENT);
            getWindow().setStatusBarColor(Color.TRANSPARENT);

        }

        itemView1.setOnClickListener(this);
    }

    @Override
    public void onClick(View v) {

    }
}

File: app/src/main/java/com/zs/login/myvarioustest/activity/ItemActivity.kt
package com.zs.login.myvarioustest.activity

import android.app.Activity
import android.os.Bundle
import android.view.View
import butterknife.ButterKnife
import com.zs.login.myvarioustest.R
import com.zs.login.myvarioustest.util.bindView
import com.zs.login.myvarioustest.view.ItemView
import org.jetbrains.anko.toast

/**
 * Created by zs
 * Date：2017年 05月 19日
 * Time：15:58
 * —————————————————————————————————————
 * About:
 * —————————————————————————————————————
 */

class ItemActivity : Activity(), View.OnClickListener {

    val itemView1: ItemView by bindView(R.id.item_1)

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.item_layout)
        ButterKnife.bind(this)

//        if (Build.VERSION.SDK_INT >= 21) {
//            val decorView = window.decorView
//            val option = View.SYSTEM_UI_FLAG_LAYOUT_HIDE_NAVIGATION or View.SYSTEM_UI_FLAG_LAYOUT_FULLSCREEN or View.SYSTEM_UI_FLAG_LAYOUT_STABLE
//            decorView.systemUiVisibility = option
//            window.navigationBarColor = Color.TRANSPARENT
//            window.statusBarColor = Color.TRANSPARENT
//
//        }
        itemView1!!.setOnClickListener(this)

    }

    override fun onClick(v: View) {

        when(v.id){
            R.id.item_1 ->{
                itemView1.content = "ssssss"
                toast("item1")
            }

        }

    }
}
